src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -235,16 +231,17 @@ internal class DeleteExpiredDocumentsCommandDto : TransactionOperationsMerger.IR
     {
         public ExpiredDocumentsCleaner.DeleteExpiredDocumentsCommand ToCommand(DocumentsOperationContext context, DocumentDatabase database)
         {
-            var expired = new Dictionary<Slice, List<(Slice LowerId, string Id)>>(SliceComparer.Instance);
+            var queue = new Queue<ExpirationStorage.DocumentExpirationInfo>();
             foreach (var item in Expired)
             {
-                expired[item.Key] = item.Value;
+                queue.Enqueue(new ExpirationStorage.DocumentExpirationInfo(item.Item1.Clone(context.Allocator), item.Item2.Clone(context.Allocator), item.Item3));
```

Comment:
Can we use named tuple so we won't have `.Item1` and  `.Item2` here?

Reply:
done